docs/sftt/sftt-api-spec.md

```diff
@@ -232,15 +232,15 @@ Create a team. The team will only contain the member that created it who is now
 
 ##### `200 OK`
 
-!!! missing "Should return the same as the get team route"
+Returns the same response as the get teams route.
 
 ##### `400 BAD REQUEST`
 
-!!! missing "Should be an error here if the team name is already taken"
+If the team name is taken.
```

Comment:
We'd also throw this is an email was invalid right? In that end we should also specify the type in the JSON body as EMAIL instead of STRING